doc/rst/source/greenspline.rst

```diff
@@ -26,7 +26,7 @@ Synopsis
 [ |-S|\ **c\|t\|l\|r\|p\|q**\ [*pars*] ] [ |-T|\ *maskgrid* ]
```

Comment:
-G should be moved to required section for consistency

Reply:
OK done, and fixed -D via new SYN for those modules using it.